src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -97,11 +114,12 @@ protected virtual Task ProcessSingleRouteAsync(HttpContext context, DownstreamRo
     /// Processing the downstream routes (no route keys).
     /// </summary>
     /// <param name="context">The main http context.</param>
-    /// <param name="route">The route.</param>
-    private async Task ProcessRoutesAsync(HttpContext context, Route route)
+    /// <param name="route">The route.</param>
+    /// <param name="bufferedBody">The buffered body.</param>
+    private async Task ProcessRoutesAsync(HttpContext context, Route route, Stream bufferedBody)
```

Comment:
Stream bufferedBody everywhere...

Reply:
Yeah that's the drawback for a more general use.